.circleci/config.yml

```diff
@@ -232,14 +232,14 @@ jobs:
             npm link ../../dist
             npm run test-types
             npm test
-      # - run:
-      #     name: check if webpack cli works well with load-pyodide.js
-      #     command: |
-      #       git clone https://github.com/pyodide/pyodide-webpack-example.git
-      #       cd pyodide-webpack-example
-      #       npm ci
-      #       cp ../src/js/*.js node_modules/pyodide/
-      #       npx webpack
+      - run:
+          name: check if webpack cli works well with load-pyodide.js
+          command: |
+            git clone https://github.com/pyodide/pyodide-webpack-example.git
+            cd pyodide-webpack-example
+            npm ci
+            cp ../src/js/*.js node_modules/pyodide/
```

Comment:
So I think the issue was that that package specifies pyodide 0.18.0 https://github.com/pyodide/pyodide-webpack-example/blob/main/package.json#L18
while we want to use it with latest pyodide. So this was a workaround for that, but after migrating to typescript, I don't think that's correct anymore?